src/object.cr

```diff
@@ -676,7 +676,7 @@ class Object
         \{% if name.is_a?(TypeDeclaration) %}
           {{var_prefix}}\{{name.var.id}} : \{{name.type}}?
 
-          def {{method_prefix}}\{{name.var.id}}? : \{{name.type}}?
+          def {{method_prefix}}\{{name.var.id}}? : \{{name.type}}
```

Comment:
This is technically a breaking change because currently `getter? foo : String { nil }` is valid. But I'd consider that a bug.